ddtrace/context.py

```diff
@@ -26,7 +32,8 @@
         _MetaDictType,  # _meta
         _MetricDictType,  # _metrics
     ]
-
+# regex matches ",", ";", ":", and characters outside the ASCII range 0x20 to 0x7E
+_TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
```

Comment:
```suggestion
_W3C_TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
```

without the prefix it sounds like a datadog feature